Jenkins_setup.md

```diff
@@ -142,6 +146,52 @@ The teardown script is entered into the Jenkins Post-Build Actions dialog like:
 
 ![Jenkins Build dialog](pictures/jenkins_post_build.png)
 
+## Setting up a bare metal Jenkins Slave Node
+
+### Setting up a jenkins user
+
+Generally, Jenkins Master needs to log into the slave node as `jenkins` user to run the
+all Jenkins job.
+So you need to create a new user `jenkins` on your slave node and add this new user to
+the `sudo` group, which grants it `sudo` priviledges.
+
+```
+$ sudo adduser jenkins
+$ sudo usermod -aG sudo jenkins
+```
+
+Not only just having sudo priviledges, the `jenkins` user also needs to be able to run `sudo`
+without a password.
+Log into your slave node as root and open the `/etc/sudoers` file with `visudo` editor.
+> **Note** You should never edit `/etc/sudoers` with a regular text editor, such as `Vim`
+> or `nano`, because they do not validate the syntax like the `visudo` editor.
+```
+sudo visudo
+```
+Add following line at the end of the `/etc/sudoers` file.
+```
+jenkins  ALL=(ALL) NOPASSWD:ALL
+```
+
+### Providing ssh private keys
+
+We need your host ssh private keys, which will lodge in the Jenkins master private key store.
```

Comment:
- s/ssh/SSH/
- s/will lodge/will be stored/ ?

Reply:
I'd probably rewrite this to drop the 'we' etc....

---

You will need to set up an SSH key-pair for the `jenkins` user on your Jenkins slave, and provide the SSH private key to the Kata Containers CI administrators to be stored in the Jenkins master secure key store.

---

OK?

Reply:
super cool for poor-english me. ;)